src/import/ImportMP3_MPG123.cpp

```diff
@@ -374,6 +348,31 @@ ProgressResult MP3ImportFileHandle::Import(WaveTrackFactory *trackFactory,
    return mUpdateResult;
 }
 
+bool MP3ImportFileHandle::SetupOutputFormat()
+{
+   long rate;
+   int channels;
+   int encoding = MPG123_ENC_FLOAT_32;
+   mpg123_getformat(mHandle, &rate, &channels, &encoding);
```

Comment:
This funcition has a return value and can fail, is that a redundant check?

Reply:
We call mpg123_getformat after parsing the first frame, so it will not fail as per the source and docs.